### DISCUSSION TITLE
have anyone tried to deploy it on macos?

### DISCUSSION THREAD

**Discussion by snfxgsx** [TheAlphamerc/flutter_twitter_clone#159] [Q&A]
I've got these messages on xcode:

Showing Recent Messages
/Users/snfxgsx/Desktop/flutter_twitter_clone-master/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths

/Users/snfxgsx/Desktop/flutter_twitter_clone-master/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths

/Users/snfxgsx/Desktop/flutter_twitter_clone-master/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths

/Users/snfxgsx/Desktop/flutter_twitter_clone-master/ios/Flutter/Debug.xcconfig:1: could not find included file 'Pods/Target Support Files/Pods-Runner/Pods-Runner.debug.xcconfig' in search paths

**Comment by TheAlphamerc** [comment]
I didn't tried it on Macos but on iOS it works fine.
